.github/workflows/workflow.yml

```diff
@@ -56,9 +60,11 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: ${{ env.IMAGE_NAME }}
-          flavor: latest=true
+          # For better caching, images can be tagged with «latest» tag
+          # when the branch is the default branch.
           tags: |
-            type=sha,prefix={{branch}}-
+            type=raw,value=latest,enable=${{ env.IS_DEFAULT_BRANCH }}
```

Comment:
Does it make sense to add `latest` by default since we are not able to enable caching?

Enabling caching would make repositories using our boilerplate required to wire up a registry immediately.

Reply:
I guess so? What is the difference between `flavor: latest = true` and `value=latest` in the tags?

Reply:
We're only adding it in default branch, which means we can use this workflow for other branches (but still have cache)